src/test/java/com/commercetools/sync/cartdiscounts/CartDiscountSyncOptionsBuilderTest.java

```diff
@@ -0,0 +1,273 @@
+package com.commercetools.sync.cartdiscounts;
+
+import com.commercetools.sync.commons.utils.TriFunction;
+import io.sphere.sdk.cartdiscounts.CartDiscount;
+import io.sphere.sdk.cartdiscounts.CartDiscountDraft;
+import io.sphere.sdk.cartdiscounts.CartDiscountDraftBuilder;
+import io.sphere.sdk.cartdiscounts.commands.updateactions.ChangeName;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.commands.UpdateAction;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import java.util.function.Function;
+
+import static io.sphere.sdk.models.LocalizedString.ofEnglish;
+import static java.util.Collections.emptyList;
+import static java.util.Collections.singletonList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+public class CartDiscountSyncOptionsBuilderTest {
+
+    private static final SphereClient CTP_CLIENT = mock(SphereClient.class);
+    private CartDiscountSyncOptionsBuilder cartDiscountSyncOptionsBuilder =
+        CartDiscountSyncOptionsBuilder.of(CTP_CLIENT);
+
+    @Test
+    public void of_WithClient_ShouldCreateCartDiscountSyncOptionsBuilder() {
+        final CartDiscountSyncOptionsBuilder builder = CartDiscountSyncOptionsBuilder.of(CTP_CLIENT);
```

Comment:
You can use here the global instance attribute `cartDiscountSyncOptionsBuilder` at line 33 instead of defining a new one.